### ISSUE TITLE
Adjust footer to add alt text to the logo

### ISSUE THREAD

**Issue by erussey** [emory-libraries/aspace#142]
Adjust the footer on all pages so that the logo has alt text for accessibility requirements.

**Comment by maxdmayhew** [comment]
title and role to improve accessibility
 
![Screenshot 2023-05-12 at 10.48.27 AM.png](https://images.zenhubusercontent.com/633dacae52c9e27764e5849b/b0d431c8-9cd0-4fac-a78f-257560de9884)

submitting PR

**Comment by maxdmayhew** [comment]
https://github.com/emory-libraries/aspace_static_plugins/pull/52

**Comment by maxdmayhew** [comment]
changed to Libraries!

**Comment by maxdmayhew** [comment]
![Screenshot 2023-05-12 at 10.53.51 AM.png](https://images.zenhubusercontent.com/633dacae52c9e27764e5849b/4425ecab-ad2a-4fd2-b6f7-7bacb814cc7a)

**Comment by erussey** [comment]
I do not know how to test this?

**Comment by erussey** [comment]
@maxdmayhew : You've got a misspelling in the alt text.  It should be "Emory Libraries and shield logo."

**Comment by maxdmayhew** [comment]
woops! PR submitted.

https://github.com/emory-libraries/aspace_static_plugins/pull/55
